test/SlowTests/Issues/RavenDB-22491.cs

```diff
@@ -83,52 +87,49 @@ private class User
         public string Name { get; set; }
     }
 
-    private class GetRevisionsSizeOperation : IMaintenanceOperation<GetRevisionsSizeOperation.Results>
+    private class GetRevisionsSizeOperation : IMaintenanceOperation<RevisionSizeDetails>
     {
-        private readonly GetRevisionsSizeParameters _parameters;
-
-        public class Results
-        {
-            public RevisionSizeDetails[] Sizes { get; set; }
-        }
+        private string _changeVector;
 
-        public static readonly Func<BlittableJsonReaderObject, Results> ToResults = JsonDeserializationClient.GenerateJsonDeserializationRoutine<Results>();
+        private static readonly Func<BlittableJsonReaderObject, RevisionSizeDetails> ToResults = JsonDeserializationClient.GenerateJsonDeserializationRoutine<RevisionSizeDetails>();
 
-        public GetRevisionsSizeOperation(List<string> changeVectors)
+        public GetRevisionsSizeOperation(string changeVector)
         {
-            _parameters = new GetRevisionsSizeParameters { ChangeVectors = changeVectors };
+            _changeVector = changeVector;
         }
 
-        public RavenCommand<Results> GetCommand(DocumentConventions conventions, JsonOperationContext context)
+        public RavenCommand<RevisionSizeDetails> GetCommand(DocumentConventions conventions, JsonOperationContext context)
         {
-            return new GetRevisionsMetadataAndMetricsCommand(_parameters);
+            return new GetRevisionsMetadataAndMetricsCommand(_changeVector);
         }
 
-        private class GetRevisionsMetadataAndMetricsCommand : RavenCommand<Results>
+        private class GetRevisionsMetadataAndMetricsCommand : RavenCommand<RevisionSizeDetails>
         {
-            private readonly GetRevisionsSizeParameters _parameters;
+            private string _changeVector;
+
 
-            public GetRevisionsMetadataAndMetricsCommand(GetRevisionsSizeParameters parameters)
+            public GetRevisionsMetadataAndMetricsCommand(string changeVector)
             {
-                _parameters = parameters;
+                _changeVector = changeVector;
             }
 
             public override bool IsReadRequest { get; }
 
             public override HttpRequestMessage CreateRequest(JsonOperationContext ctx, ServerNode node, out string url)
             {
-                url = $"{node.Url}/databases/{node.Database}/revisions/size";
+                url = $"{node.Url}/databases/{node.Database}/revisions/size?changeVector={UrlEncode(_changeVector)}";
```

Comment:
use `Uri.EscapeDataString` instead of `UrlEncode`.
the change vector is a base64, which can contain `+` char, which seems to have some bad treatment when using `UrlEncode` among other things (https://stackoverflow.com/a/37110034)